packages/devtools_app/lib/src/timeline/timeline_flame_chart.dart

```diff
@@ -151,6 +150,13 @@ class FullTimelineFlameChartCanvas extends FlameChartCanvas<FullTimelineData> {
           maxZoomLevel: 40000,
         );
 
+  static const guidelineWidth = 0.4;
+
+  static const guidelineColor = ThemedColor(
```

Comment:
can we share the same color constants used for the guide lines in the inspector instead of adding an additional color constant? I'm open to tweaking the inspector guide line color if it doesn't work for your use case due to the different background colors you have.
For example, we could tweak the alpha of the colors for the inspector if that is needed to make them play better with the varying background colors in this view.

Reply:
sg